front_end/content.py

```diff
@@ -330,6 +379,41 @@ def get_user_count(self):
         self.cursor.execute(sql)
         return self.cursor.fetchone()["count"]
 
+    def check_course_exists(self, course_id):
+        sql = '''SELECT COUNT(*) AS count
+                 FROM courses
+                 WHERE course_id = ?'''
+        self.cursor.execute(sql, (course_id,))
+        if self.cursor.fetchone():
+            return True
+        else:
+            return False
+
+    def get_course_from_role(self, user_id):
```

Comment:
This doesn't seem to be related to roles specifically. Should you have a role in the WHERE clause? Please make the function name more specific, too.